code_samples/sql-basic-usage.js

```diff
@@ -19,8 +19,12 @@ const { Client, SqlColumnType } = require('hazelcast-client');
 
 (async () => {
     try {
-        const client = await Client.newHazelcastClient();
-        const mapName = 'myMap';
+        const client = await Client.newHazelcastClient({
+            properties: {
+                'hazelcast.logging.level': 'OFF'
```

Comment:
There is no change in the logging stuff in this PR, but should there be a logging setting instead of the property? Also why is the logging turned off?

Reply:
Does the logging clutter the output that much? What happens if the user forgets to run a cluster?

Reply:
Maybe I should revert this change. It will, a little bit. But if we want to do it we should do it for all code samples I guess. Reverting..

> There is no change in the logging stuff in this PR, but should there be a logging setting instead of the property?

There is a customLogger setting at the root of config object but not logging level. Maybe it is better have it in a more convinient place. However, adding another config should deprecate this one.  @mdumandag  What do you think?